pkg/ddl/db_test.go

```diff
@@ -1195,6 +1195,17 @@ func TestAdminAlterDDLJobUnsupportedCases(t *testing.T) {
 	tk.MustGetErrMsg("admin alter ddl jobs 1 thread = 257;", "the value 257 for thread is out of range [1, 256]")
 	tk.MustGetErrMsg("admin alter ddl jobs 1 batch_size = 31;", "the value 31 for batch_size is out of range [32, 10240]")
 	tk.MustGetErrMsg("admin alter ddl jobs 1 batch_size = 10241;", "the value 10241 for batch_size is out of range [32, 10240]")
+	tk.MustGetErrMsg("admin alter ddl jobs 1 max_write_speed = '2PiB';", "the value 2PiB for max_write_speed is out of range [0, 1125899906842624]")
```

Comment:
some error message cc @lance6716

Reply:
The negative tests cases are not enough. Please check at least max_write_speed = 1.23, max_write_speed = RAND(), max_write_speed = 30+40, and maybe ask other reviewers to come up with more cases.

Reply:
add some cases and fix comment in https://github.com/pingcap/tidb/pull/57611/commits/d71a82654c90cea79b1a36f3fc1565afcf4a99ac，PTAL again